src/components/MoneyReportHeader.js

```diff
@@ -126,6 +129,17 @@ function MoneyReportHeader({session, personalDetails, policy, chatReport, report
                         />
                     </View>
                 )}
+                {shouldShowSubmitButton && !isSmallScreenWidth && (
+                    <View style={styles.pv2}>
+                        <Button
+                            medium
+                            success={moneyRequestReport.isOwnPolicyExpenseChat}
```

Comment:
Lets also have a fallback here in case the property is missing
```suggestion
                            success={chatReport.isOwnPolicyExpenseChat}
```